PaaConditionalMenu/src/paaconditionalmenu/PaaConditionalMenu.java

```diff
@@ -0,0 +1,86 @@
+package paaconditionalmenu;
+
+import java.util.Scanner;   // Needed for Scanner class
+import java.util.Date;      // Needed for Date and Time
+import java.text.DateFormat;    // Needed for Date and Time
+import java.text.SimpleDateFormat;  // Needed for Date and Time
+import java.util.Calendar;      // Needed for Date and Time
+
```

Comment:
It is similar to the method java doc I talked about go anywhere above the __class__ statement and type /** and hit return. Ctrl space will show you the kind of @ annotations you can add.